### DISCUSSION TITLE
What are the full contents of config:recommended? Documentation docs.renovatebot.com/presets-config/ doesn't mention that it contains :prHourlyLimit2

### DISCUSSION THREAD

**Discussion by bzakdd** [renovatebot/renovate#28069] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github

### Please tell us more about your question or problem

I am using `renovate/renovate:latest` docker image.

I reading what configuration is contained in `config:recommended`. As a source of truth I used [documentation](https://docs.renovatebot.com/presets-config/#configrecommended) which maps `config:recommended` to:
```json
{
  "extends": [
    ":dependencyDashboard",
    ":semanticPrefixFixDepsChoreOthers",
    ":ignoreModulesAndTests",
    "group:monorepos",
    "group:recommended",
    "replacements:all",
    "workarounds:all"
  ]
}
```
I've looked at all of those presets quickly:
- :dependencyDashboard
- :semanticPrefixFixDepsChoreOthers
- :ignoreModulesAndTests
- group:monorepos
- group:recommended
- replacements:all
- workarounds:all

And none of them extends the `:prHourlyLimit2`.

However I see empirically that `config:recommended` extends `:prHourlyLimit2`. Even [documentation](https://docs.renovatebot.com/configuration-options/#ignorepresets) confirms that (in the chapter explaining `ignorePresets`): 
```json
{
  "extends": ["config:recommended"],
  "ignorePresets": [":prHourlyLimit2"]
}
```

What are the full contents of `config:recommended`?

**Comment by rarkins** [comment, marked as answer]
The example you found is outdated, because `config:recommended` no longer extends `:prHourlyLimit2`, because Renovate _defaults_ to 2 PRs per hour without needing any presets. See https://docs.renovatebot.com/configuration-options/#prhourlylimit

**Reply by bzakdd** [reply]
Thank you very much!
